src/psyclone/transformations.py

```diff
@@ -2265,7 +2265,7 @@ def name(self):
         '''
         return "OCLTrans"
 
-    def apply(self, sched, opencl=True):
+    def apply(self, sched, opencl=True, options={}):
```

Comment:
I should have picked this up first time around but pylint warns that `{}` is a dangerous default value. Please use None instead and change the method body appropriately.

Reply:
Fixed